docs/team/markusyeo.md

```diff
@@ -13,19 +13,25 @@ Given below are my contributions to the project.
 
 
 * **Enhancements implemented:**
-    * to be added soon
+  * **New Feature**: Changed default file storage format to be CSV
+    * Justification: As we deal with tabular data, we would like users to be able to easily view the data entered into `DengueHotspotTracker` offline. 
```

Comment:
Can also add that NEA personnel are more likely to be familiar with CSV than json!

Reply:
Got it!